### DISCUSSION TITLE
Cyberduck Mac Version 8.6.3 (40040) keeps asking for "System" keychain access for 17 times each time I login

### DISCUSSION THREAD

**Discussion by gascarrier** [iterate-ch/cyberduck#15149] [Q&A] [answered]
**Describe the bug**
Cyberduck Mac Version 8.6.3 (40040) keeps asking for "System" keychain access for 17 times each time I login

**To Reproduce**
Every time I login to a ftp, or sftp, or ftps server. 

**Expected behavior**
Do not show "System" keychain access window.

**Screenshots**
![image](https://github.com/iterate-ch/cyberduck/assets/5868963/654dccdc-ab63-450f-98c6-7319b5de3975)

**Desktop (please complete the following information):**
 - OS: MacOS 
 - Version: 13.5.2 (22G91)
 - CyberDuck version: 8.6.3 (40040)

**Comment by dkocher** [comment]
This indicates you have saved a password for the server in your _System_ keychain instead of the _Login_ keychain. Open _Keychain Access.app_ and choose _System_ in the sidebar to locate any passwords saved there.

**Comment by ejtasker** [comment, marked as answer]
I discovered the source of this issue for me was an "untrusted" "identity preference" for the ftp server in my keychain. I fixed this inside the "Keychain Access.app".
